mesonbuild/mesonlib/universal.py

```diff
@@ -1950,7 +1950,7 @@ def get_wine_shortpath(winecmd: T.List[str], wine_paths: T.Sequence[str],
             os.unlink(bat_file.name)
     wine_path = ';'.join(rel_paths + wine_paths)
     if len(wine_path) > 1024:
-        mlog.warning('WINEPATH exceeds 1024 characters which could cause issues')
+        mlog.warning('WINEPATH exceeds 1024 characters which could cause issues', fatal=False)
```

Comment:
Should the user fix this by moving the worktree, and thus maybe want it to error out?

Reply:
I have no idea, TBH. The comment is "this could cause issues" seems to imply that in some cases it's fine, in others it's not and we don't konw, so if it fails you can figure it out.

Reply:
I'll just leav it as fatal and we can change it later if we need to